inspirehep/modules/records/cli.py

```diff
@@ -133,12 +130,6 @@ def get_query_records_to_index(pid_types):
             PersistentIdentifier.pid_type.in_(pid_types),
             PersistentIdentifier.object_type == 'rec',
             PersistentIdentifier.status == PIDStatus.REGISTERED,
-            or_(
-                not_(
-                    type_coerce(RecordMetadata.json, JSONB).has_key('deleted')
-                ),
-                RecordMetadata.json["deleted"] == cast(False, JSONB)
-            )
```

Comment:
is the `join` still needed now you don't need to access the record any more?

Reply:
yep